src/types.rs

```diff
@@ -156,6 +156,15 @@ pub struct CrateVersion {
     pub dependencies: Vec<Dependency>,
 }
 
+impl CrateVersion {
+    /// Parse and return the semantic version of this crate as semver::Version
```

Comment:
Let's also mention the probability (14 in 2_000_000?) at which this method may panic.